package.json

```diff
@@ -25,6 +25,7 @@
     "@angular/router": "^4.0.0",
     "core-js": "^2.4.1",
     "material-design-icons": "^3.0.1",
+    "ng2-dnd": "^4.2.0",
```

Comment:
Drag and drop para angular 2+, muyyyy util y facil de usar, intente manualmente pero era un camello.